### PR TITLE
Add version:bump and version:show tasks for version-bump-bot

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -27,60 +27,60 @@ require_relative "tasks/external_tests"
...
-  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
-    config.future_release = Chef::VERSION
-    config.enhancement_labels = "enhancement,Enhancement,New Feature,Feature".split(",")
-    config.bug_labels = "bug,Bug,Improvement,Upstream Bug".split(",")
-    config.exclude_labels = "duplicate,question,invalid,wontfix,no_changelog,Exclude From Changelog,Question,Discussion".split(",")
-  end
-rescue LoadError
-  puts "github_changelog_generator is not available. gem install github_changelog_generator to generate changelogs"
-end
+# task :pedant, :chef_zero_spec
```
Comment (coderanger) [inline Rakefile]: Erm, assuming something is in progress?

Comment (jkeiser) [inline Rakefile]: I have no idea what you are talking about! Look, there aren't even any changes to the `Rakefile`!

(Thanks, updated :)

File: chef-config/lib/chef-config/package_task.rb [hunk 2]
```diff
@@ -185,10 +190,27 @@ def define
...
+        task :bump => %w{version:bump_patch version:update}
+
+        task :show do
+          puts version
+        end
+
+        # Add 1 to the current patch version in the VERSION file, and write it back out.
+        task :bump_patch do
+          current_version = version
+          new_version = current_version.sub(/^(\d+\.\d+\.)(\d+)/) { "#{$1}#{$2.to_i + 1}" }
```
Comment (mwrock) [inline chef-config/lib/chef-config/package_task.rb]: not a blocker but using Gem::Version to parse the patch and increment might be more clear.

**Comment by jkeiser** [comment]
:+1:

**Comment by mwrock** [comment]
:+1:

File: chef-config/lib/chef-config/package_task.rb (hunk 2 above)
Comment (jkeiser) [inline chef-config/lib/chef-config/package_task.rb]: It was silly, but I was actually trying to keep it as a string to keep the file identical in all respects except the patch version :)

**Pull request merged**